polaris-icons/icons/ActivityLog.yml

```diff
@@ -0,0 +1,12 @@
+name: Activity log
+description: N/A
+keywords:
+  - Page
+  - Log
+  - Activity
+authors:
+  - José Torre
+version: 1
```

Comment:
If we do chose to remove the version (#11127), we should remember to update these here as well 👍